docs/community_resources/package_structure.md

```diff
@@ -88,21 +88,25 @@ A package may introduce a namespace for contracts, to be consumed by first and t
 and projects, which must be prefixed as:
 
 ```php
-namespace Ibexa\Platform\Contracts;
+namespace Ibexa\Contracts;
 ```
 
 Examples:
 
 ```php
-namespace Ibexa\Platform\Contracts\Kernel;
+namespace Ibexa\Contracts\Kernel;
 ```
 
 ```php
-namespace Ibexa\Platform\Contracts\SiteFactory;
+namespace Ibexa\Contracts\SiteFactory;
 ```
 
 ```php
-namespace Ibexa\Platform\Contracts\Commerce\Shop;
+namespace Ibexa\Contracts\Commerce\Shop;
 ```
 
 That namespace needs to be mapped to the `src/contracts` directory of a package.
+
+!!! note
+
+    Backward compatibility for interfaces and objects defined in the `Contracts` namespace is guarranteed.
```

Comment:
Should we add something along the lines: "Compatibility for other code is maintained on a best effort basis"?

Reply:
Isn't it somethin that is already assumed for the whole product? That is why we maintain BC breakdown lists, mention BC promises etc.

Reply:
>> Should we add something along the lines: "Compatibility for other code is maintained on a best effort basis"?

> Isn't it something that is already assumed for the whole product? That is why we maintain BC breakdown lists, mention BC promises etc.

It is assumed but history shows not very clear for 3rd party Consumers. Worth to state it clear. I would also include here SemVer (sth like Backward compatibility according to [SemVer](https://semver.org/) rules or some similar sentence rephrased). Otherwise it might create an expectation of never ending BC.

Reply:
Symfony has a good description for it I guess?

https://symfony.com/doc/current/contributing/code/bc.html#:~:text=That's%20why%20we%20promise%20you,allowed%20to%20break%20backward%20compatibility.

Reply:
> Symfony has a good description for it I guess?
> 
> https://symfony.com/doc/current/contributing/code/bc.html#:~:text=That's%20why%20we%20promise%20you,allowed%20to%20break%20backward%20compatibility.

This is Symfony promise, not ours. Let's not suggest it's the same, because we have no control over it.

Reply:
That's not what I meant :smile: I just wanted to use their explanation as a base for ours.

Reply:
That is a great idea for a larger declaration of our philosophy. We know that a promise is needed, but we want to wait until the details crystallize, as far as what we can actually promise. This PR is only about reassuring our clients about interfaces and objects defined in the `Contracts` namespace.